homeassistant/components/tautulli/__init__.py

```diff
@@ -1 +1,66 @@
-"""The tautulli component."""
+"""The Tautulli integration."""
+from __future__ import annotations
+
+from pytautulli import PyTautulli
+
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY, CONF_SSL, CONF_URL, CONF_VERIFY_SSL
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DATA_KEY_COORDINATOR, DEFAULT_NAME, DOMAIN
+from .coordinator import TautulliDataUpdateCoordinator
+
+PLATFORMS = [SENSOR_DOMAIN]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Tautulli from a config entry."""
+    api_client = PyTautulli(
+        api_token=entry.data[CONF_API_KEY],
+        url=entry.data[CONF_URL],
+        session=async_get_clientsession(hass, entry.data[CONF_VERIFY_SSL]),
+        verify_ssl=entry.data[CONF_VERIFY_SSL],
+        ssl=entry.data[CONF_SSL],
+    )
+    cordnator = TautulliDataUpdateCoordinator(hass, api_client)
+    await cordnator.async_config_entry_first_refresh()
+
+    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = {DATA_KEY_COORDINATOR: cordnator}
```

Comment:
We don't store anything else in hass.data, set the coordinator directly, no need for DATA_KEY_COORDINATOR here